build.gradle.kts

```diff
@@ -34,7 +34,7 @@ tasks.withType<ShadowJar> {
     doLast {
         copy {
             from("src/main/kotlin/org/example/BTreeBot.json")
-            from("$buildDir/libs/BTreeBot.jar")
```

Comment:
this is string interpolation. It's a variable so i wouldn't hardcode it to "build"

Reply:
is this $buildDir be set  somewhere?
my issue was, when i follow the instruction, the $buildDir is not set.

Reply:
it has a default value: https://docs.gradle.org/current/userguide/writing_build_scripts.html I cloned it on a fresh machine and it builds fine for me.

Reply:
reverted